antenna-documentation/src/site/markdown/generators/attribution-document-generator-step.md

```diff
@@ -0,0 +1,39 @@
+# Attribution Document generator
+
+### How to use
+Add this configuration to the workflow.xml
+
+```xml
+<workflow>
+    <generators>
+        <step>
+            <name>Attribution Document Generator</name>
+            <classHint>org.eclipse.sw360.antenna.attribution.document.workflow.generators.AttributionDocumentGenerator</classHint>
+            <configuration>
+                <entry key="attribution.doc.templateKey" value="basic-pdf-template"/>
+                <entry key="attribution.doc.name" value=""/>
+                <entry key="attribution.doc.productName" value=""/>
+                <entry key="attribution.doc.productVersion" value=""/>
+                <entry key="attribution.doc.copyrightHolder" value=""/>
+            </configuration>
+        </step>
+    </generators>
+</workflow>
+```
+
+#### Explanation of parameters
+* `attribution.doc.templateKey`: Specifies the bundle, which will be used for the generation of the attribution document.
```

Comment:
If I understand this correctly, it still is a must. Why not make it optional, and the default is hardcoded in Antenna?

Reply:
Good point. After adding the basic template, I forgot to change it in the AttributionDocumentGenerator class.